tailwind.config.js

```diff
@@ -0,0 +1,201 @@
+/** @type {import('tailwindcss').Config} */
+const { BREAKPOINT_REM } = require('./src/constants/page');
+const plugin = require('tailwindcss/plugin');
+
+export default {
+  content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+  theme: {
+    screens: {
+      mobile: { max: BREAKPOINT_REM.mobile },
+      notMobile: { min: BREAKPOINT_REM.mobile },
+      tablet: { max: BREAKPOINT_REM.tablet },
+      notTablet: { min: BREAKPOINT_REM.tablet },
+      desktopSmall: { max: BREAKPOINT_REM.desktopSmall },
+      desktopMedium: { min: BREAKPOINT_REM.desktopMedium },
+      desktopLarge: { min: BREAKPOINT_REM.desktopLarge },
+    },
+    colors: {
+      white: 'var(--color-white)',
+      black: 'var(--color-black)',
+      green: 'var(--color-green)',
+      red: 'var(--color-red)',
+
+      'white-faded': 'var(--color-white-faded)',
+
+      'layer-0': 'var(--color-layer-0)',
+      'layer-1': 'var(--color-layer-1)',
+      'layer-2': 'var(--color-layer-2)',
+      'layer-3': 'var(--color-layer-3)',
+      'layer-4': 'var(--color-layer-4)',
+      'layer-5': 'var(--color-layer-5)',
+      'layer-6': 'var(--color-layer-6)',
+      'layer-7': 'var(--color-layer-7)',
+
+      border: 'var(--color-border)',
+      'border-white': 'var(--color-border-white)',
+      'border-red': 'var(--color-border-red)',
+
+      'text-0': 'var(--color-text-0)',
+      'text-1': 'var(--color-text-1)',
+      'text-2': 'var(--color-text-2)',
+      'text-button': 'var(--color-text-button)',
```

Comment:
textColor would be more descript, so usage becomes `text-textColor-0` instead of `text-text-0`